assets/admin/js/globalpayments-admin.js

```diff
@@ -81,6 +84,73 @@
             return $( this.getLiveModeSelector() ).is( ':checked' );
         },
 
+        /**
+         * Checks if gateway setting is enabled
+         *
+         * @returns {*|jQuery}
+         */
+        isEnabled: function() {
+            return $( this.getEnabledGatewaySelector() ).is( ':checked' );
+        },
+
+        /**
+         * Checks if cc_types at least one selected
+         */
+        validate_cc_types: function () {
+            if ( 'globalpayments_googlepay' != this.id) {
```

Comment:
@apetrovici  I had to add this for the validation of the multicheckbox for google pay.